examples/resources/bloxone_infra_join_token/resource.tf

```diff
@@ -0,0 +1,3 @@
+resource "bloxone_infra_join_token" "example" {
+  name = "example_join_token"
```

Comment:
Can we add another example where we also add the fields `description `, `expires_at `, and `tags`?